docs/07-web-development.Rmd

```diff
@@ -350,8 +350,52 @@ It is not a perfect solution, but it should be very rare that you run into probl
 
 ## Useful resources
 
+### Optimization
+
 Although\index{Optimization} static websites are fast in general, you can certainly further optimize them. You may search for "CSS and JavaScript minifier", and these tools can compress your CSS and JavaScript files, so that they can be loaded faster. Since there are a lot of tools, I will not recommend any here.
 
 You can also optimize images on your website. I frequently use a command-line tool named [`optipng`](http://optipng.sourceforge.net) to optimize PNG images. It is a lossless optimizer, meaning that it reduces the file size of a PNG image without loss of quality. From my experience, it works very well on PNG images generated from R, and can reduce the file size by at least 30% (sometimes even more than 50%). Personally I also use online tools like http://optimizilla.com to optimize PNG and JPEG images. For GIF images, I often use https://ezgif.com/optimize to reduce the file sizes if they are too big.
 
 Note that Netlify has provided the optimization features on the server side for free at the moment, so you may just want to enable them there instead of doing all the hard work by yourself.
+
+### blogdown resources
```

Comment:
I think this subsection is much more essential than the rest. I'd mention the example blogdown-demo.rbind.io in Section 1.5 instead.

Reply:
I tried to find a good place in 1.5 for this- it was a little harder than I thought and am worried the link there will be hard to discover?